### ISSUE TITLE
Development Mode

### ISSUE THREAD

**Issue by Gimenz** [michellexliu/receiptify#3]
are this works with spotify dev with status in Development mode ?

**Comment by tashima42** [comment]
Yes, it works with Spotify Dev mode, but only to manually added users on the "Users and Access" section
